src/components/Navbar.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Navbar = () => (
+  <div className="navbar">
+    <h1>Math Magicians</h1>
+    <div className="nav-links">
+      <a href="/">Home</a>
+      <a href="/calculator">Calculator</a>
+      <a href="/quote">Quote</a>
+    </div>
+  </div>
+);
```

Comment:
Look at the image below
- [ ] As you know we are trying to build a Single-Page-Application (SPA) with React. SPA typically means that our application should not **"reload"** in the traditional sense. Your browser should not refresh when I try to navigate to another page but it does. This is because you are using the **`a`** tag for your Navbar links. But **`react-router-dom`** has **`NavLink`** and **`Link`** features for proper navigation without browser reload. Kindly replace your **`a`** with **`NavLink`** or **`Link`** and ensure that your browser does not reload.

  You can refer to this [article](https://medium.com/swlh/using-react-router-navlink-to-specify-the-active-element-in-a-navigation-bar-38700ffd4900) on how to do that.

 
![lufte-reload](https://user-images.githubusercontent.com/36542986/144223293-25e0e0d7-3481-4df3-be5c-dd507e7424ad.jpg)

Reply:
solved this

Reply:
solved this